### ISSUE TITLE
[Multi-sig] feat: quality of life improvements on the multi-sig branch

### ISSUE THREAD

**Issue by bassgeta** [JoinColony/colonyCDapp#2658]
### Story

This is a catch-all issue for all minor improvements on the multi-sig branch.

### Description

List of improvements needed:
1. After multisig is installed, invalidate the query for decision methods, so you can immediately create an action - @mmioana 
2. Install multisig and immediately give it permissions so we don't need the `Enable permissions banner` @mmioana 
3. After users are given multi-sig permissions (including when the extension is installed and as a result of a manage permissions action / motion finalization), invalidate the permission holder query so we can immediately create multi-sig actions with them as potential signers @mmioana 
4. Unify types and how our `getMotionParams` helpers in the sagas work like
5. Go over and remove any useless TODO comments
6. Add rejected at date to the `MultiSig` widget @mmioana 
7. The MultiSig SVG isn't according to the designs @mmioana 
8. Rework `VoteButton` and `RemoveVoteButton` to not use `ActionForm` but use `ActionButton` instead (loader is an issue) @bassgeta 
9. Instead of `isMultiSig?: boolean;` , maybe we can define a literal type here so like: `motionType: 'multiSig' | 'reputation' | 'someOtherMotionType'`
10. Check the Permissions page is showing users correctly with all filter options @iamsamgibbs - I investigated this, there are issues, but there are also issues on master so have opened a new issue to resolve it there first (#2784, #2785) Will need fixing up here after these are resolved
11. Refactor `createEditDomainReputationMotion` saga to use updated `getChildIndex` and `getPermissionsProof` functions & fix "Latest activity" bug for domain action related entries @rumzledz 
12. When the extension is enabled, users should be assigned multi-sig roles which reflect their current permission roles. For example, if Amy has custom permissions in Serenity to have the funding and architecture roles, after multi-sig has installed, she should have funding and architecture multi-sig roles in Serenity. (Update this in the `handleMultiSigInstall` function.) - this can be a bigboi, opened its own issue for it [here](https://github.com/JoinColony/colonyCDapp/issues/2765) 
13. Once #2565 is merged into main, rebase feat/multi-sig and check it is still working as expected, but that it also works in a sensible way with the new multi-sig authority. (There are a couple of other PRs kicking around at the moment which touch this form too)
14. ~~Rejecting as the motion owner should automatically cancel the motion (I thought PagePro were handling this as part of the UI issue, so double check this with Kerry first)~~ @iamsamgibbs 
15. Remove multi-sig meatball component (and any references to it) all multi-sig actions should just use the regular meatball component on the completed action screen (double check this is being included on all multi-sig motions). @mmioana 
16. If motion is older than a week, call `finalize` instead of `finalizeWithoutFailure` @bassgeta 
17. Disable multisig for unsupported action types (manage tokens, payment builder) @bassgeta 
18. ~~Clean up finalize rejection now that motion auto rejects when reaching the rejection threshold~~ @iamsamgibbs 
19. Comment out automatic role assignment as it is now, and tackle that when doing [2765](https://github.com/JoinColony/colonyCDapp/issues/2765) @bassgeta 
20. ~~After [this](https://github.com/JoinColony/colonyCDapp/pull/2767) is merged, figure out why do multiSig roles always show up for the colony owner even though they aren't in the backend~~ @iamsamgibbs 
![Image](https://github.com/user-attachments/assets/fa72ad95-8ec4-426b-8fa9-25dd2ad2c233)

~~Also, roles are inherited by users with core permissions, but not for users with only multi-sig permissions:~~

<img width="395" alt="Screenshot 2024-07-25 at 09 39 15" src="https://github.com/user-attachments/assets/bea3c0f6-c910-48b7-9dd0-cd1b333965b8">

21. The "not enough members" notice does not show until after this has finished fetching. If I had filled out this form in order before selecting "multi-sig" then I would easily have enough time to submit the form before the message shows. We could maybe add the threshold check as part of the form validation too? @bassgeta 

https://github.com/user-attachments/assets/6cab29b7-5430-4bca-a43a-d130d72c61dc

This might also be related - the "invalid threshold" blip is visible even when not throttling the connection:

https://github.com/user-attachments/assets/f492c03c-11d1-408a-9234-a5a3a290c230

22. ~~You need to refresh the extension page after updating the threshold. This is something we should also clear the cache on (or whatever needs to be done to fix it) - after doing some testing, only clearing the cache might not be enough, given the extension threshold might not yet be saved. So probably we need to update the getExtensionByColonyAndHash query similar to how we do it in `utils/members.ts > updateContributorQueries`~~ @iamsamgibbs 
23. The domain threshold is not looked up correctly (set a subdomain's threshold to 1, the global one to 2, and try to create a reputation smite in the subdomain). We need to compare the domain's nativeId instead. @bassgeta 
![image](https://github.com/user-attachments/assets/ed26f241-9183-40c5-9ff0-d272bff73ab7)
![image](https://github.com/user-attachments/assets/f2649244-43ff-4f96-872f-16bcbe6ebe63)

**Comment by rumzledz** [comment]
I added a new entry (#9) but not entirely sure if this will be valid based on the future architecture of motions.

**Comment by mmioana** [comment]
I will pick the first point ftm ✨ 
After multisig is installed, invalidate the query for decision methods, so you can immediately create an action

**Comment by iamsamgibbs** [comment]
Closing out as agreed with @bassgeta as pretty much everything here has been tackled
